src/lib/viewers/controls/settings/SettingsDropdown.tsx

```diff
@@ -74,52 +71,69 @@ export default function SettingsDropdown<V extends Value = string>({
 
     useClickOutside(dropdownElRef, () => setIsOpen(false));
 
+    // Customize the forwarded ref to combine usage with the ref internal to this component
+    React.useImperativeHandle(
```

Comment:
It seems unorthodox to expose the button as the component's ref, since it's a compound component. If we want to expose the dropdown itself later, that could get confusing.